backend/opampd/protocol.go

```diff
@@ -56,3 +67,31 @@ func (p *Protocol) OnAgentDisconnect(instanceUid string, disconnect *protobufs.A
 	//TODO implement me
 	return nil, fmt.Errorf("implement me")
 }
+
+func (p *Protocol) createEntity(instanceUid string, report *protobufs.StatusReport) error {
+	entity := &corev2.Entity{
+		EntityClass:        corev2.EntityAgentClass,
```

Comment:
I don't think it's very important, but I think we said we would create a new entity class altogether, like `OT Agent` or something.

Reply:
It does matter for filtering etc.